src/index.ts

```diff
@@ -29,7 +29,7 @@ const ensureUrlEndsWithSlash = (url: string) => {
  * then Danger will comment with a warning on the pull request asking the developer
  * to include the JIRA issue identifier in the pull request title.
  */
-export default function jiraIssue(options: Options) {
+export default function jiraIssue(options?: Options) {
```

Comment:
This is a tricky one - the way the plugin currently works, the user must always pass an `Options` object in. The `options || {}` check is deceiving in the sense that if a `dangerfile.js` user doesn't pass an object in, I wanted to properly throw an error telling them they are missing a required option (since they don't have the compile time type checking benefit that `dangerfile.ts` users have).

I think changing `options` to be optional would then allow a user to not pass an `Options` object in, which would be OK at TypeScript compile time but would always throw an Error at runtime.

Suggestions on how to better document or type this in TypeScript? 🤔